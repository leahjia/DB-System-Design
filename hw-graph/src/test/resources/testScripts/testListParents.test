# University Map parents of Node testing file.
# Tests that a list of sources that can directly reach a node

# Finds the parents of node A
ListParents Node A

# Finds the parents of node B
ListParents Node B