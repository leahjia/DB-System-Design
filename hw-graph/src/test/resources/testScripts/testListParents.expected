# University Map parents of Node testing file.
# Tests that a list of sources that can directly reach a node

# Finds list of sources that directly connects to node A
Found all parents of node A

# Finds list of sources that directly connects to node B
Found all parents of node B
